Gemfile

```diff
@@ -3,3 +3,5 @@
 source "https://rubygems.org"
 
 gemspec
+
+gem "jekyll", "~> 4.3"
```

Comment:
Is this an artifact of your system or a necessary upgrade?

Reply:
Seems to be an artifact, it runs fine with or without it. I added a commit to get rid of it.